.github/workflows/build-docker-monorepo.yml

```
@@ -0,0 +1,95 @@
+      - '.github/versions.sh'
+      - '.github/workflows/build-docker-monorepo.yml'
+  pull_request:
+    paths:
+      - 'tools/docker/Dockerfile.monorepo'
+      - 'tools/docker/bin/monorepo'
+      - '.github/versions.sh'
+      - '.github/workflows/build-docker-monorepo.yml'
+concurrency:
+  group: build-docker-monorepo-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build and publish Jetpack Monorepo Environment
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: matticbot
+          password: ${{ secrets.DOCKER_HUB_MATTICBOT_TOKEN }}
+
+      - name: Log in to GitHub Packages
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Fetch build args
+        id: buildargs
+        run: |
+          source .github/versions.sh
+          source .github/files/gh-funcs.sh
+
+          gh_set_output php-version "$PHP_VERSION"
+          gh_set_output composer-version "$COMPOSER_VERSION"
+          gh_set_output node-version "$NODE_VERSION"
+          gh_set_output pnpm-version "$PNPM_VERSION"
+
+          if [[ "$GITHUB_EVENT_NAME" == "push" ]]; then
+            gh_set_output tags "type=raw,latest"
+            gh_set_output images $'automattic/jetpack-monorepo\nghcr.io/automattic/jetpack-monorepo'
+          elif [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
+            gh_set_output tags "type=ref,event=pr"
+            gh_set_output images "ghcr.io/automattic/jetpack-monorepo"
+          else
+            echo "Unknown GITHUB_EVENT_NAME $GITHUB_EVENT_NAME"
+            exit 1
+          fi
+
+      - name: Extract Docker metadata
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          flavor: latest=false
+          tags: ${{ steps.buildargs.outputs.tags }}
+          images: ${{ steps.buildargs.outputs.images }}
+          labels: |
+            org.opencontainers.image.title=Jetpack Monorepo Environment
+            org.opencontainers.image.description=Environment for building and testing the Jetpack Monorepo.
+            org.opencontainers.image.documentation=${{ github.server_url }}/${{ github.repository }}/blob/trunk/tools/docker/README.md
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: tools/docker
+          file: tools/docker/Dockerfile.monorepo
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          platform: linux/arm64,linux/amd64
```

```suggestion
          platforms: linux/arm64,linux/amd64
```
You may also need to re-add this bit from build-docker.yml
https://github.com/Automattic/jetpack/blob/0dd5ba80f3356d54203d8bf5fc5802ddf01268ca/.github/workflows/build-docker.yml#L34-L37
(if not, then we can probably remove the bit from there too 🤷)